### PR TITLE
Submission Error Handling

### PR DESCRIPTION
**Pull request by aka-jain**
@deshraj  @spyshiv : please review!

### CODE REVIEW

File: frontend/src/js/controllers/challengeCtrl.js [hunk 1]
```diff
@@ -219,44 +219,84 @@
...
-                                                // vm.input_file.name = '';
 
-                                            angular.forEach(
-                                                angular.element("input[type='file']"),
-                                                function(inputElem) {
-                                                    angular.element(inputElem).val(null);
-                                                }
-                                            );
+                                    var fileVal = angular.element(".file-path").val();
+                                    console.log(fileVal)
```
Comment (deshraj) [inline frontend/src/js/controllers/challengeCtrl.js]: Please remove the debug message.

File: frontend/src/js/controllers/challengeCtrl.js [hunk 2]
```diff
@@ -219,44 +219,84 @@
...
+                                                vm.phaseId = null;
+                                                vm.subErrors.msg = "Your submission has been recorded succesfully!"
+                                                vm.stopLoader();
+                                            },
+                                            onError: function(response) {
+                                                var status = response.status;
+                                                var error = response.data;
+
+                                                vm.phaseId = null;
+                                                // console.log(error)
```
Comment (deshraj) [inline frontend/src/js/controllers/challengeCtrl.js]: Please remove this line.

File: frontend/src/views/web/challenge/submission.html [hunk 3]
```diff
@@ -4,6 +4,16 @@
         <div class="col s12 m12">
             <div class=" ev-card-panel ev-z-depth-5 ">
                 <!-- <div class=" ev-card-panel ev-z-depth-5"> -->
+                <div class="loader-container card-loader-container" id="loader" ng-show="challenge.isExistLoader" class="fade">
+                    <!-- <div class="text-dark-gray center w-300 loader-title">{{teams.loaderTitle}}</div> -->
```
Comment (deshraj) [inline frontend/src/views/web/challenge/submission.html]: Are you commenting this for future reference?

Comment (aka-jain) [inline frontend/src/views/web/challenge/submission.html]: Sure.

**Comment by aka-jain** [comment]
@deshraj : Have a look please.

**Comment by deshraj** [comment]
👀 Looks good :-)

**Pull request merged**